Userland/Libraries/LibWeb/HTML/ScrollOptions.h

```diff
@@ -15,4 +15,23 @@ struct ScrollOptions {
     Bindings::ScrollBehavior behavior { Bindings::ScrollBehavior::Auto };
 };
 
+// https://drafts.csswg.org/cssom-view/#normalize-non-finite-values
+inline double normalize_non_finite_values(double value)
```

Comment:
I think these two should either:

1. Be `[[nodiscard]]` so you can't accidentally call it without using the return value.
2. Take a `double&` and mutate the value in place.

As it is right now, it seems very easy to accidentally translate the spec to C++ as:

```cpp
// 2. Normalize non-finite values for y.
normalize_non_finite_values(y);
```

Reply:
Good point! I made it `[[nodiscard]]`